api/client/network.go

```diff
@@ -14,9 +14,26 @@ import (
 	"github.com/docker/docker/daemon/network"
 	"github.com/docker/docker/opts"
 	flag "github.com/docker/docker/pkg/mflag"
+	"github.com/docker/docker/pkg/parsers/filters"
 	"github.com/docker/docker/pkg/stringid"
+	"github.com/docker/docker/runconfig"
 )
 
+// supportedFilters predefined some supported filter handler function
+type filterHandler func(types.NetworkResource) bool
+type filterHandlers map[string]filterHandler
+
+var supportedFilters map[string]filterHandlers
+
+func init() {
```

Comment:
you can initialize those supported filters directly when you create the variable, two lines above. This init function is not necessary.